core/domain/exp_services.py

```diff
@@ -2045,3 +2051,20 @@ def get_interaction_id_for_state(exp_id, state_name):
         return exploration.get_interaction_id_by_state_name(state_name)
     raise Exception(
         'There exist no state in the exploration with the given state name.')
+
+
+def set_exploration_edits_allowed(exp_id, edits_are_allowed):
+    """Toggled edits allowed field in the exploration.
+
+    Args:
+        exp_id: str. The ID of the exp.
+        edits_are_allowed: boolean. Whether exploration edits are allowed.
+    """
+    exploration_model = exp_models.ExplorationModel.get(exp_id)
+    exploration_model.edits_allowed = edits_are_allowed
```

Comment:
Do you think the input parameter should be validated here to be a boolean (and throw an error otherwise)?